internal/wiz/structs.go

```diff
@@ -562,10 +562,13 @@ type AzureServiceBusAutomationActionParams struct {
 	QueueURL                string    `json:"queueUrl"`
 }
 
-// Connector struct -- updates.  this resource is incomplete.  missing ConnectorConfigs ConnectorModules TenantIdentityClient
+// Connector struct -- updates.  this resource is incomplete
+// not all ConnectorConfigs are defined, missing ConnectorModules TenantIdentityClient
 type Connector struct {
-	AddedBy           User            `json:"addedBy"`
-	AuthParams        json.RawMessage `json:"authParams"`
+	AddedBy           User            `json:"addedBy"` // this has been deprecated and should be replaced by CreatedBy
+	AuthParams        interface{}     `json:"authParams"`
+	Config            interface{}     `json:"config,omitempty"`  // type assertion i.e. ConnectorConfigAWS
+	Outpost           interface{}     `json:"outpost,omitempty"` // type assertion i.e. OutpostAWSConfig
```

Comment:
This shows as type Outpost in the GraphQL schema.  Why is it an interface{} in golang?